src/dashboard/apps/consent/forms.py

```diff
@@ -123,3 +133,14 @@ class ConsentForm(forms.Form):
             },
         ),
     )
+
+    def __init__(self, *args, **kwargs):
+        """Initialize the form."""
+        entity = kwargs.get("initial", None).pop("entity", None)
+
+        super().__init__(*args, **kwargs)
+
+        if entity:
+            self.fields["contract_holder_name"].initial = entity.contract_holder_name
+            self.fields["contract_holder_email"].initial = entity.contract_holder_email
+            self.fields["contract_holder_phone"].initial = entity.contract_holder_phone
```

Comment:
Not a big fan of this pattern, I'd rather provide appropriate initial field values given an entity in your view.